react/graphql/RecipeList.graphql

```diff
@@ -0,0 +1,7 @@
+query RecipeList($appName: String, $category: String) {
+  getRecipeList(appName: $appName, category: $category) {
```

Comment:
👀 `get`

Reply:
Hahahaha I knooow, I promise I'm gonna rename my queries, but I need to release a major for `vtex.docs-graphql` to do so.